deployment/aws/tests/performance-tests/ec2-userdata-scripts/run-gaffer-random-generator.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash -xe
+
+# Install required software
+yum remove -y java-1.7.0-openjdk
+yum install -y java-1.8.0-openjdk-devel
+
+MAVEN_VERSION=3.5.0
+MAVEN_DOWNLOAD_URL=http://www.mirrorservice.org/sites/ftp.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+function install_dev_tools {
+	# Install git
+	if ! which git >/dev/null 2>&1; then
+		yum install -y git
```

Comment:
Some yum installs you do as the current user and some you do with sudo. Should they all be done with sudo?

Reply:
Some scripts are run by a local user and some are run as root - which is the case here. But sure, I can run all yum commands via sudo to be consistent.